sale_layout_category_hide_detail/models/sale_order.py

```diff
@@ -13,9 +13,11 @@ class SaleOrderLine(models.Model):
     show_subtotal = fields.Boolean(
         string="Show subtotal",
         default=True)
+    show_price = fields.Boolean(string="Show Price", default=True)
```

Comment:
```suggestion
    show_price = fields.Boolean(
        string="Show Price",
        default=True)
```
To match the existing code style